plugins/pulp_rpm/plugins/profilers/yum.py

```diff
@@ -347,7 +359,7 @@ def _rpms_applicable_to_consumer(consumer, errata_rpms):
         return applicable_rpms, older_rpms
 
     @staticmethod
-    def _translate_erratum(unit, repo_ids, consumer, conduit):
+    def _translate_erratum(unit, repo_ids, consumer, conduit, available_rpm_nevras):
```

Comment:
ugh, i remember thinking "i will add this later" when writing it. Good catch, thx.